<ui version="4.0" >
 <comment>MultiMeterSettingsWidget</comment>
 <class>MultiMeterSettingsWidget</class>
 <widget class="QWidget" name="MultiMeterSettingsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Title:</string>
       </property>
       <property name="buddy" >
        <cstring>m_title</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="m_title" >
       <property name="whatsThis" >
        <string>Enter the title of the display here.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_showUnit" >
       <property name="whatsThis" >
        <string>Enable this to append the unit to the title of the display.</string>
       </property>
       <property name="text" >
        <string>&amp;Show unit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="m_lowerLimitActive" >
       <property name="whatsThis" >
        <string>Enable the minimum value alarm.</string>
       </property>
       <property name="text" >
        <string>&amp;Enable alarm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_lblLowerLimit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Lo&amp;wer limit:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>m_lowerLimit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="m_lowerLimit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="m_upperLimitActive" >
       <property name="whatsThis" >
        <string>Enable the maximum value alarm.</string>
       </property>
       <property name="text" >
        <string>E&amp;nable alarm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_lblUpperLimit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Upper limit:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>m_upperLimit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="m_upperLimit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Normal digit color:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KColorButton" name="m_normalDigitColor" >
       <property name="text" >
        <string/>
       </property>
       <property name="color" >
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel2_2" >
       <property name="text" >
        <string>Alarm digit color:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="KColorButton" name="m_alarmDigitColor" >
       <property name="text" >
        <string/>
       </property>
       <property name="color" >
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="textLabel3_2" >
       <property name="text" >
        <string>Background color:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="KColorButton" name="m_backgroundColor" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kcolorbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_upperLimitActive</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_upperLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>396</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_upperLimitActive</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_lblUpperLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lowerLimitActive</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_lowerLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>396</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lowerLimitActive</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_lblLowerLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
